### ISSUE TITLE
Dashboard A11y: Focus on preview close

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#4205]
## Bug Description

Focus is lost on close of the preview dialog. 

## Expected Behaviour

Focus should be returned to the story that was selected for preview. 

## Steps to Reproduce

Go to the dashboard and preview a story or a template (requires a feature flag to be enabled), close the preview and see that focus has disappeared. 

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by BrittanyIRL** [comment]
WIP pr is here: https://github.com/google/web-stories-wp/pull/4338 to focus back on the grid. getting to the item is proving more of a time suck, will circle back as time allows.

**Comment by csossi** [comment]
Testing in QA Environment - PR 4424 (Win10/Chrome)

Work as described fine for Story previews, but not for Templates. Not getting the "preview" popup after enabling the flag for template previews
